### DISCUSSION TITLE
i have a great idea for comboscaling rebalance

### DISCUSSION THREAD

**Discussion by IdkeMint** [ppy/osu#21529] [Ideas] [answered]
gameplay should have some sections for diffrent skillsets for example you have 1k combo and 900 from this 1k combo was on stream sections and 100 was on jumps sections, now imagine you breaking combo on jamp part and your combo from jump section(you'll not see this it's fantom and only for good pp combo scaling), and what actualy will happen, you'll break combo only on jump section and this will be insensibly if you play map based on streams and you won't have to retry maps for getting pp only if you breaking combo on small sections you will not lose all your pp combo

**Comment by ghost** [comment]
currently there is no way to get the location of a sliderbreak or miss for use in pp calculation

**Reply by IdkeMint** [reply]
i think there's not problem to add it on laser

**Comment by apollo-dw** [comment, marked as answer]
Scaling misses based on the difficulty of the missed section definitely sounds like something we'd want when it's possible.

**Reply by IdkeMint** [reply]
thanks for your reply)
